docker/README.md

```diff
@@ -0,0 +1,37 @@
+# toutv in a Docker container
+It's just a Docker image for https://github.com/bvanheu/pytoutv
+
+## How to use
+Very similarly to the pytoutv client, except you don't have to install python3 nor pip3 nor toutv scripts and their dependencies, everything is provided in this image.
+
+Here are a few examples on how to use this image , [list of commmands is provided on pytoutv Github repo](https://github.com/bvanheu/pytoutv) : 
+
+```
+$ docker run anthonydahanne/toutv
```

Comment:
That example would have to change if you address the comment below.

Reply:
true